config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    suggestion: ''
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_FAILED:
+    step: 'install-update: deploy-static-content'
+    title: 'Command `/bin/magento setup:static-content:deploy` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_COMPRESSION_FAILED:
+    step: compress-static-content
+    title: 'Static content compression failed'
+    suggestion: 'Check the `cloud.log` for more information.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_CANNOT_UPDATE_VERSION:
+    step: 'deploy-static-content:generate'
+    title: 'Cannot update the deployed version'
+    suggestion: 'Cannot update `./pub/static/deployed_version.txt` file. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SCD_CLEAN_FAILED:
+    step: clean-static-content
+    title: 'Failed to clean static content files'
+    suggestion: ''
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_SPLIT_DB_COMMAND_FAILED:
+    step: 'install-update: split-db'
+    title: 'Command `/bin/magento setup:db-schema:split` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_VIEW_PREPROCESSED_CLEAN_FAILED:
+    step: clean-view-preprocessed
+    title: 'Failed to clean the `var/view_preprocessed` folder'
+    suggestion: 'Unable to clean the `./var/view_preprocessed` folder. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_FILE_CREDENTIALS_EMAIL_NOT_WRITABLE:
+    step: 'install-update: reset-password'
+    title: 'Failed to update the `/var/credentials_email.txt` file'
+    suggestion: 'Failed to update the `/var/credentials_email.txt` file. Check your filesystem permissions.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_UPGRADE_COMMAND_FAILED:
+    step: 'install-update: update'
+    title: 'Command `/bin/magento setup:upgrade` failed'
+    suggestion: 'Check the `cloud.log` and `install_upgrade.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: deploy
+    type: critical
+!php/const Magento\MagentoCloud\App\Error::DEPLOY_CACHE_FLUSH_COMMAND_FAILED:
+    step: clean-cache
+    title: 'Command `/bin/magento cache:flush` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
```

```suggestion
    suggestion: 'Check the `cloud.log` for more information. For more detailed command output, add the `VERBOSE_COMMANDS: ''-vvv''` option to the `.magento.env.yaml` file.'
```

If this wording works for you, replace all occurences in file.